mobify/mobify-code-style
# Slack publish (was "npm auto publish")

Linked PRs: N/A

## Changes
- ~~Sets up CircleCI to automatically publish to npm on merges to `master`~~
- Configures project so it can be released using Mobitron (https://github.com/mobify/mobitron/pull/124)
- Sets up doc linting on `README.md` and `RELEASE.md`

## How To Test
- Do a release and merge to `master`
- Monitor CircleCI build and ensure it publishes the package to npm

## Applicable Research Resources
- N/A

## TODOs:

~~- [ ] Updated CHANGELOG~~
